### PR TITLE
Add Permit Launch Week

### PR DESCRIPTION
**Pull request by gemanor**
## What kind of change does this PR introduce?

Add Permit Launch Week

## What is the current behavior?

Missing link

## What is the new behavior?

Added link

### CODE REVIEW

**Review by fmerian — CHANGES_REQUESTED** [review]
thanks for the suggestion, @gemanor! they are duplicate links tho. I'd keep the `Go to launch page` only.

File: README.md [hunk 1]
```diff
@@ -40,8 +41,9 @@ For best practices, read this:
 
 ### 2024
 
-> Total count: 51
+> Total count: 52
 
+- 2024 / 10 / 28-01: Permit Launch Week. [Teaser ↗︎](https://launchweek.permit.io) [Go to launch page ↗︎](https://launchweek.permit.io)
```
Comment (gemanor) [inline README.md:46]:
```suggestion
- 2024 / 10 / 28-01: Permit Launch Week [Teaser ↗︎](https://x.com/permit_io/status/1847213978352545944) (https://launchweek.permit.io) [Go to launch page ↗︎](https://launchweek.permit.io)
```

File: README.md [hunk 2]
```diff
@@ -4,6 +4,7 @@
 
 > You? [Create an issue](https://github.com/supabase-community/launchweek.dev/issues) or open a [Pull Request (PR)](https://github.com/supabase-community/launchweek.dev/pulls) to add your upcoming launch week here. Enjoy the fun!
 
+- 2024 / 10 / 28-01: Permit Launch Week. [Teaser ↗︎](https://launchweek.permit.io) [Go to launch page ↗︎](https://launchweek.permit.io)
```
Comment (gemanor) [inline README.md:7]:
```suggestion
- 2024 / 10 / 28-01: Permit Launch Week. [Teaser ↗︎](https://x.com/permit_io/status/1847213978352545944) [Go to launch page ↗︎](https://launchweek.permit.io)
```

**Comment by gemanor** [comment]
Thanks for the review @fmerian, just fixed it.

**Review by fmerian — APPROVED** [review]
LGTM :)

**Pull request merged**